src/force/potential.cu

```diff
@@ -100,14 +100,20 @@ static __global__ void gpu_find_force_many_body(
 
       // per-atom virial
       s_sxx += x12 * f21x;
+      s_syy += y12 * f21y;
```

Comment:
This function is for non-NEP potentials, and should  be brought back to its original form.

Reply:
Here some virial components are double accumulated.

Reply:
Ah okay, I've restored it!